src/libraries/System.Console/ref/System.Console.cs

```diff
@@ -69,10 +69,10 @@ public static partial class Console
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("ios")]
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("tvos")]
         public static bool TreatControlCAsInput { get { throw null; } set { } }
-        public static int WindowHeight { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("android"), System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser"), System.Runtime.Versioning.UnsupportedOSPlatformAttribute("ios"), System.Runtime.Versioning.UnsupportedOSPlatformAttribute("tvos")] get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
+        public static int WindowHeight { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("android"), System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser"), System.Runtime.Versioning.UnsupportedOSPlatformAttribute("ios"), System.Runtime.Versioning.UnsupportedOSPlatformAttribute("tvos")] get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows"), System.Runtime.Versioning.SupportedOSPlatformAttribute("linux"), System.Runtime.Versioning.SupportedOSPlatformAttribute("freebsd"), System.Runtime.Versioning.SupportedOSPlatformAttribute("osx")] set { } }
```

Comment:
@buyaa-n @stephentoub So far, getting windows height and width was working on all OSes except browser, ios, adnroid and tvos.
Setting the value was working only on Windows. So the attributes applied were like this:

```cs
public static int WindowHeight
{ 
    [UnsupportedOSPlatformAttribute("android")]
    [UnsupportedOSPlatformAttribute("browser")]
    [UnsupportedOSPlatformAttribute("ios")]
    [UnsupportedOSPlatformAttribute("tvos")]
    get { throw null; } 
    [SupportedOSPlatformAttribute("windows")]
    set { } }
}
```

With this PR, both sets of supported/unsupported OSes are the same:
- supported: Windows, Linux, macOS, FreeBSD
- unsupported: Android, iOS, tvOS, Broweser

The question: what is the recommended way to annotate the APIs in such case:
a) marking getter and setter of WindowHeight as unsupported on the mentioned platforms (a removal of `[SupportedOSPlatformAttribute("windows")]` from the setter)

```cs
[UnsupportedOSPlatformAttribute("android")]
[UnsupportedOSPlatformAttribute("browser")]
[UnsupportedOSPlatformAttribute("ios")]
[UnsupportedOSPlatformAttribute("tvos")]
public static int WindowHeight
{ 

    get { throw null; } 
    set { } }
}
```

b) Extending the list of supported OSes for the setter (no breaking changes, but using two different ways of expressing the same thing for one public property which can be confusing)

```cs
public static int WindowHeight
{ 
    [UnsupportedOSPlatformAttribute("android")]
    [UnsupportedOSPlatformAttribute("browser")]
    [UnsupportedOSPlatformAttribute("ios")]
    [UnsupportedOSPlatformAttribute("tvos")]
    get { throw null; } 
    [SupportedOSPlatformAttribute("windows")]
    [SupportedOSPlatformAttribute("linux")]
    [SupportedOSPlatformAttribute("freebsd")]
    [SupportedOSPlatformAttribute("osx")]
    set { } }
}
```

Reply:
```cs
[UnsupportedOSPlatformAttribute("android")]
[UnsupportedOSPlatformAttribute("browser")]
[UnsupportedOSPlatformAttribute("ios")]
[UnsupportedOSPlatformAttribute("tvos")]
```
Looks cleaner though it means the API is unsupported on android, browser, ios and tvOS, but supported all other, not only windows, linux, freebsd, osx.  Not sure if that is OK.

For the PCA analyzer it doesn't really matter where the attribute applied, but not sure if it's moving the attributes is OK for API compat side

Reply:
Is there a downside to option (a)?  That seems like the clear winner :)

Reply:
@adamsitnik @buyaa-n I think I need help on https://github.com/dotnet/runtime/pull/75824/commits/e315d16aae366e45b9a416664ed54b2f5c36840f
How do I fix `CP0014` error?